website/docs/eks/java/eks-create.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Creating an Amazon EKS Cluster
+sidebar_position: 1
+---
+
+## Objective
+
+Creating an Amazon EKS cluster with [eksctl](https://eksctl.io/) allows for a wide range of configurations to cater to different needs. This lab shows you how to create an Amazon EKS cluster using [eksctl](https://eksctl.io/) and various command line tools.
+
+How does Amazon EKS work?
+
+![what-is-eks](./images/what-is-eks.png)
+
+- Create an Amazon EKS cluster in the AWS Management Console or with the AWS CLI or one of the AWS SDKs.
+- Launch managed or self-managed Amazon EC2 nodes, or deploy your workloads to AWS Fargate.
+- When your cluster is ready, you can configure your favorite Kubernetes tools, such as kubectl, to communicate with your cluster.
+- Deploy and manage workloads on your Amazon EKS cluster the same way that you would with any other Kubernetes environment. You can also view information about your workloads using the AWS Management Console.
+
+## Prerequisites
+
+- [Building and Running Multi-Architecture Containers](../../containers/java/upload-ecr.md)
+
+[Amazon Elastic Kubernetes Service (Amazon EKS)](https://aws.amazon.com/eks/) is a managed service that you can use to run Kubernetes on AWS without needing to install, operate, and maintain your own Kubernetes control plane or nodes. Kubernetes is an open-source system for automating the deployment, scaling, and management of containerized applications.
+
+## 1. Running the UnicornStore with Amazon EKS
+
+Below you can find a visualization on how the UnicornStore application will be deployed and communicate with Amazon EKS:
+
+![unicornstore-architecture-eks](./images/unicornstore-architecture-eks.png)
```

Comment:
Just a comment that can be fixed later on.
This image shows incoming requests received by ALB and sent to aws-lb-controller. Actually, aws-lb-controller is not in the network flow, ALB sends requests directly to App pod/container (or K8s svc).
Users are often confused thinking that network traffic goes through the aws-lb-controller, which is not correct.